docs/src/examples/atomix.md

```diff
@@ -0,0 +1,69 @@
+# Atomic operations with Atomix.jl
+
+In case the different kernels access the same memories, [race conditions](https://en.wikipedia.org/wiki/Race_condition) can occur.
+KA is compatible with [Atomix.jl](https://github.com/JuliaConcurrent/Atomix.jl) to avoid this.
+
+## Race conditions
+
+The following goes wrong:
+```julia
+using CUDA, KernelAbstractions, CUDA.CUDAKernels, Atomix
+using ImageShow, ImageIO
+
+
+function index_fun(arr; backend=CUDABackend())
```

Comment:
```suggestion
function index_fun(arr; backend=get_backend(arr))
```

Reply:
Ah; I didn't know of `get_backend`!
Thanks, that helps my code too :)